modules/reference/pages/feature/Jakarta-EE9-feature-updates.adoc

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2021 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+// https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+// IBM Corporation
+//
+// This doc is hosted in the Red Hat Runtimes documentation. Any changes made to this doc also need to be made to the version that's located in the PurpleLiberty GitHub repo (https://github.com/PurpleLiberty/docs).
+//
+:page-description: For Jakarta EE 9 applications to run in Open Liberty, new Jakarta EE 9 features must be specified in the server configuration. If an old name is incorrectly used with a new Jakarta EE 9 version, the server does not start, and the server logs provide information to say which new feature name must be used.
+:seo-title: Jakarta EE9 feature updates - OpenLiberty.io
+:projectName: Open Liberty
+:page-layout: Jakarta EE9
+:page-type: feature updates
+= Jakarta EE9 feature updates
+
+For Jakarta EE 9 applications to run in Open Liberty, new Jakarta EE 9 features must be specified in the server configuration. If an old name is incorrectly used with a new Jakarta EE 9 version, the server does not start, and the server logs provide information to say which new feature name must be used.
```

Comment:
avoid using "new/old" as it will go out of date as the doc gets older. This isn't really a concern in the blogs but for docs it is